RackApp/app.rb

```diff
@@ -0,0 +1,85 @@
+require 'erb'
+require 'rack'
+require 'active_support/all'
```

Comment:
We do not need to require all active_support. Always require only the necessary library if it is small